### DISCUSSION TITLE
ControlNet finetuning

### DISCUSSION THREAD

**Discussion by ofzlo** [PixArt-alpha/PixArt-alpha#12] [Q&A] [answered]
Hi, thanks for sharing your awesome work.

Could you give information about ControlNet fine-tune dataset or the process of finetuning?
I wonder how did you access to HED edge image dataset.

**Comment by lawrence-cj** [comment, marked as answer]
You may use existing HED algorithm on any dataset you desire to get the HED data. In our paper, we use [HED](https://github.com/lllyasviel/ControlNet-v1-1-nightly/blob/main/annotator/hed/__init__.py) on JourneyDB for training.

**Reply by ofzlo** [reply]
thanks for your support!

**Reply by ofzlo** [reply]
How many images did you used? Then how did you sampling the dataset? randomly?

**Reply by lawrence-cj** [reply]
Around 3M HED data for ControlNet training with random sampling strategy.
